src/components/dialogs/stashed-elements/stashed-elements-dialog.tsx

```diff
@@ -27,6 +27,7 @@ interface IStashedElementsDialog {
     onClose: () => void;
     stashedElements: any[];
     onStashedElementChange: () => any[];
+    activeDirectory?: any;
```

Comment:
```suggestion
    activeDirectory?: IDirectory;
```
This would fit I guess

Reply:
done